kitodo-mediaserver-core/src/main/resources/config/default.yml

```diff
@@ -92,7 +108,8 @@ importer:
   workIdRegex: "[\\w-]+"
   cron: 0 5 2 * * *
   indexWorkAfterImport: true
-
+  notificationEmail:
+    - admin@example.org
```

Comment:
In default.yml the email lists should be empty as you can not override it in local.yml.

Reply:
Ah ok, i could then comment it out since I want to show a hint on how to use it.